vagrant/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- import_playbook: hostname.yml
+- import_playbook: centos.yml
+- import_playbook: postgres.yml
+  #- import_playbook: playbooks/solr.yml
```

Comment:
do we need these commented out entries or could we remove them?

Reply:
They can be removed.